configs/config.go

```diff
@@ -0,0 +1,52 @@
+package configs
+
+import (
+	"io/ioutil"
+	"log"
+	"time"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+const ConfigurationPath = "/etc/gz-beat-shipper/gz-beat-shipper-conf.yml"
+
+type Configuration struct {
+	Host     string `yaml:"host"`
+	Port     string `yaml:"port"`
+	Path     string `yaml:"path"`
+	Registry string `yaml:"registry"`
+	Recheck  string `yaml:"recheck"`
+}
+
+func (c *Configuration) GetConfiguration() *Configuration {
```

Comment:
You can leave this function as 'void', not much use for its returned value.

Reply:
You do nothing with the return value of this function. I am saying that you could return nothing altogether.